packages/adapter-prisma/src/index.ts

```diff
@@ -157,7 +157,45 @@ import type { Adapter, AdapterAccount, AdapterAuthenticator, AdapterSession, Ada
  * ```
  *
  * Everything else should be the same.
- *
+ * 
+ * ### Edge runtime compatibility
+ * At the moment, Prisma is still working on being fully compatible with the Vercel edge runtime. See the issue being tracked [here](https://github.com/prisma/prisma/issues/20560). There are two options to deal with this issue:
+ * - Use the Prisma's [Accelerate plan](https://pris.ly/d/accelerate)
+ * - Switch to the `jwt` session strategy. Below is an example:
```

Comment:
```suggestion
 * At the moment, Prisma is still working on being fully compatible with the Vercel edge runtime. See the issue being tracked [here](https://github.com/prisma/prisma/issues/20560), and Prisma’s announcement about edge support in early access since 5.9.0 [here](https://github.com/prisma/prisma/releases/tag/5.9.0). There are two options to deal with this issue:
 * 
 * - Use the Prisma’s [Accelerate](https://pris.ly/d/accelerate) feature
 * - Switch to the jwt session strategy
 * 
 * Using Prisma in and edge runtime with `jwt` session strategy looks like this.
```